lib/ansible/modules/clustering/kubevirt/kubevirt_pvc.py

```diff
@@ -0,0 +1,204 @@
+    - If openshift >= 0.6.2, this defaults to C(['strategic-merge', 'merge']), which is ideal for using the same parameters
+      on resource kinds that combine Custom Resources and built-in resources. For openshift < 0.6.2, the default
+      is simply C(strategic-merge).
+    choices:
+    - json
+    - merge
+    - strategic-merge
+    type: list
+  name:
+    description:
+      - Use to specify a PVC object name.
+    required: true
+    type: str
+  namespace:
+    description:
+      - Use to specify a PVC object namespace.
+    required: true
+    type: str
+  annotations:
+    description:
+      - Annotations attached to this object.
+      - U(https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+    type: dict
+  labels:
+    description:
+      - Labels attached to this object.
+      - U(https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
+    type: dict
+  selector:
+    description:
+      - A label query over volumes to consider for binding.
+      - U(https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
+    type: dict
+  access_modes:
+    description:
+      - accessModes contains the desired access modes the volume should have.
+      - More info: U(https://kubernetes.io/docs/concepts/storage/persistent-volumes#access-modes)
+    type: list
+  resources:
+    description:
+      - Resources represents the minimum resources the volume should have.
+      - More info: U(https://kubernetes.io/docs/concepts/storage/persistent-volumes#resources)
+    type: dict
+  storage_class_name:
+    description:
+      - Name of the StorageClass required by the claim.
+      - More info: U(https://kubernetes.io/docs/concepts/storage/persistent-volumes#class-1)
+    type: str
+  volume_mode:
+    description:
+      - volumeMode defines what type of volume is required by the claim. Value of Filesystem is implied when not included in claim spec. This is an alpha feature of kubernetes and may change in the future.
+    type: str
+  volume_name:
+    description:
+      - volumeName is the binding reference to the PersistentVolume backing this claim.
+    type: str
+  data_transfer_type:
+    description:
+      - Upon creation, fill the volume with existing data.
+      - upload: upload data from local system. Needs to be followed up with usage of kubevirt_cdi_upload module.
+      - import: import data from remote system.
+      - clone: clone data from another PVC.
+      - TODO desc
+    choices:
+    - upload
+    - import
+    - clone
+    type: list
+  data_content_type:
+    description:
+      - Defines the content type of data being imported.
+      - I(kubevirt) denotes a virtual machine image. (Default)
+      - I(archive) denotes a tar archive.
+    choices:
+      - kubevirt
+      - archive
+  data_import_source:
+    description:
+      - Defaults to 'http' if missing of invalid.
```

Comment:
I just pasted the upstream docs for a lot of these arguments. I'll correct them for better legibility later. What they've meant here is "if you don't provide the argument or provide an invalid argument, like 'zzzzzzzzzzz'".

Reply:
Do I understand it well that if it defaults to 'http' and user would provide it empty or not at all it would end up being 'http'? This would mean that it always be there.